mappings/net/minecraft/level/biome/Biome.mapping

```diff
@@ -13,29 +13,36 @@ CLASS kd net/minecraft/level/biome/Biome
 	FIELD l NETHER Lkd;
 	FIELD m SKY Lkd;
 	FIELD n biomeName Ljava/lang/String;
-	FIELD o colour1 I
-	FIELD r colour2 I
+	FIELD o grassColour I
+	FIELD p topTileId B
+	FIELD q underTileId B
+	FIELD r foliageColour I
 	FIELD s monsters Ljava/util/List;
 	FIELD t creatures Ljava/util/List;
 	FIELD u waterCreatures Ljava/util/List;
 	FIELD v snows Z
 	FIELD w precipitates Z
 	FIELD x biomes [Lkd;
 	METHOD a createBiomeArray ()V
+	METHOD a getBiome (DD)Lkd;
+		ARG 0 x
+		ARG 2 z
 	METHOD a getSkyColour (F)I
 		ARG 1 temperature
-	METHOD a getBiome (FF)Lkd;
-		ARG 0 x
-		ARG 1 z
-	METHOD a setColour_2 (I)Lkd;
+	METHOD a getClimateBiome (FF)Lkd;
+		ARG 0 temperature
+		ARG 1 rainfall
+	METHOD a setFoliageColour (I)Lkd;
+		ARG 1 colour
 	METHOD a setName (Ljava/lang/String;)Lkd;
 		ARG 1 name
 	METHOD a getTree (Ljava/util/Random;)Lpg;
 		ARG 1 rand
 	METHOD a getSpawnList (Llk;)Ljava/util/List;
 		ARG 1 type
-	METHOD b setSnowy ()Lkd;
-	METHOD b setColour_1 (I)Lkd;
+	METHOD b snowy ()Lkd;
+	METHOD b setGrassColour (I)Lkd;
```

Comment:
```suggestion
	METHOD b grassColour (I)Lkd;
```

Reply:
it has an instance passed tho. @Blayyke what do you think

Reply:
you have `rainy` as a mutation method, so `grassColour` as a mutation method is fine

Reply:
I personally use set for methods that take a param and then set a field to that param, and no set prefix if it just sets a field to a constant ( a method sets a field to true, for example )

Reply:
:thinking:

Reply:
I'm gonna use blayyke's conventions

Reply:
discord voted for set prefixes for everything so i would say use them